source/MapSalesPanel.cpp

```diff
@@ -358,11 +363,13 @@ void MapSalesPanel::Draw(Point &corner, const Sprite *sprite, bool isForSale, bo
 		
 		DrawSprite(corner, sprite);
 		
-		const Color &textColor = *GameData::Colors().Get(isForSale ? "medium" : "dim");
+		const Color &textColor = *GameData::Colors().Get(isForSale ? "medium" : !storage.empty() ? "semidim" : "dim");
```

Comment:
In the map outfitter view, semidim makes it easy to spot which outfits are in storage but not in stock, they stand out from both "neither in stock nor in storage" (dim) and "in stock" (medium). What's more, when the yellow `Sells this outfit` filter is selected, outfits that are in storage but not in stock remain listed, and it would be odd for them to be dim.

Reply:
The outfitter can outfit both outfits that are for sale as well as outfits that are in storage. So having a different color than dim for outfits in storage makes a lot of sense to me (since dim has a feeling of meaning "not available").